pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -460,8 +459,7 @@ func maybeApplyEdsConfig(c *cluster.Cluster) {
 			ConfigSourceSpecifier: &core.ConfigSource_Ads{
 				Ads: &core.AggregatedConfigSource{},
 			},
-			ResourceApiVersion:  core.ApiVersion_V3,
-			InitialFetchTimeout: features.InitialFetchTimeout,
```

Comment:
This removes the ability to set InitialFetchTimeout=0 and not timeout?

Reply:
Yes. I will put it back. I was trying this because the credential test was failing to see whats going on. There is some thing odd with this. I will debug and put it back.